### PR TITLE
Updates to the OpenStack glossary

### PR DESCRIPTION
**Pull request by ndeevy**
Product-specific terms related to Red Hat OpenStack Platform.

Addresses #149

### CODE REVIEW

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:13]:
```suggestion
==== image:images/yes.png[yes] director (noun)
```

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:40]:
```suggestion
```

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:40]:
```suggestion
```

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:39]: Should "Exceptions" be singular if there is only a single exception of "head templates"?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:41]: I know this was already here, but would you mind changing this to "uppercase"?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:79]: So a few months ago I came across something that said "a RHOSP" in the OpenShift docs and thought that it was wrong - I assumed it would be "an" because of saying out "R-H-O-S-P". But I was told that it was actually correct to use "a RHOSP" (I can't remember why). If this is true, I think it might be worthwhile to say here. wdyt?

**Comment by bergerhoffer** [comment]
@redhat-documentation/ccs-style-council Can use a few more reviewers please!

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (sterobin) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:79]: Correct, it should be pronounced "ROsp" or "RAHsp", whichever, and therefore "a RHOSP". This is the norm for other similar cases, like RHOSAK ("ROW-sack"), RHODS ("ROads"), RHOSR ("ROsr"), etc. However, that said, I can't imagine when or why we'd ever say "a RHOSP" anyway, except like "a RHOSP instance" or "a RHOSP [something]". So maybe just gloss the pronunciation if needed. Defer to @ndeevy whether or not address it.

Comment (sterobin) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:79]: Suggest: "use _RHOSP_". Not sure why the colon.

**Comment by sterobin** [comment]
Agree with @bergerhoffer, plus a couple minor things. Otherwise looks good.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (bergerhoffer) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:79]: Thanks for the background @sterobin! To your point about not knowing when you'd use it, there is an instance of it in this PR:

```
Use director to create a RHOSP environment
```
But +1 to trying to rewrite to avoid the a/an issue, if possible.

Comment (sterobin) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:79]: @bergerhoffer Ah, sorry, missed it! Yeah, that's a perfect example. So yeah, "a RHOSP environment" is correct/compliant.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (porueesq) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:39]: The parentheses can be removed. 

```suggestion
* Use the component name instead of the code name. For example: _Block Storage_ instead of _cinder_. Exception: _heat templates_ , as opposed to _Orchestration service templates_.
```

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (porueesq) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:90]:
```suggestion
*Description*: The undercloud is the director node. It is a single-system within the RHOSP installation that includes components for provisioning and managing the RHOSP nodes that form your environment, known as the overcloud. Write in lowercase.
```

@bergerhoffer Do we include cross-references in glossaries? If we do, we can link `overcloud`

**Review by porueesq — CHANGES_REQUESTED** [review]
A couple of comments.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (ndeevy) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:41]: Oh yes! missed that. Thanks

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (bburt-rh) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:79]:
```suggestion
*Description*: On first use in a module, use the complete product name and the abbreviation in parentheses: _Red Hat OpenStack Platform (RHOSP)_. After the first instance, use: _RHOSP_. This product name applies to RHOSP 8 and later versions.
```
Can omit the article here, I think.

**Comment by porueesq** [comment]
@ndeevy Happy day! :sun_with_face:  

Can you please take a look at this PR? Let me know if there are questions.

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (bergerhoffer) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:13]: @ndeevy it looks like you removed the icons from the terms. Those should be there - can you add them back in please?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:14]: Does one pronounce RHOSP as one word or as a series of letters? I would pronounce the word in a series of letters, but this could just be me :) . 

The context of my query is around the indefinite article choice. I see an answer further down, so I'll take it RHSOP Is pronounced as one word?

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:41]:
```suggestion
* Write the code name in lowercase in parentheses after the component name, and after _service_, on the first use of the component name in a module.
```

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:40]:
```suggestion
* Write _service_ in lowercase if the component name includes _service_.
```

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (ndeevy) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:55]:
```suggestion
*Description*: The overcloud is the resulting RHOSP environment that is created by using the undercloud. Write in lowercase.
```

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (joaedwar) [inline supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:77]: @ndeevy What is the correct format for referring to products/ emphasizing products? In this guide quotation marks are sometimes used to refer or emphasize a product name (see line 85 https://github.com/redhat-documentation/supplementary-style-guide/commit/edb628bb4a145d3fbb600e794f98590b7d030f12)

Should we use quotations or italics? I think the convention is to use quotation marks for product names and italics for published book/ guide titles. WDYT?

**Review by joaedwar — COMMENTED** [review]
Looks wonderful! I just have a question about formatting

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (ndeevy) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc:77]: Thanks for bringing this up, I wasn't sure myself and went with what was there previously without giving it enough thought. From a scan through, although there are some inconsistencies, most product glossaries do use double quotations, so I'm inclined to change a lot of these on that basis.

- AMQ uses italics, like OpenStack does/did: https://redhat-documentation.github.io/supplementary-style-guide/#destination
- Ceph uses double quotations: https://redhat-documentation.github.io/supplementary-style-guide/#placement-group
- Rhel, it seems that in deployment-scanner subsystem, it's double quotes for the terminology but single for the plural 's', which makes sense to me.
- OpenShift uses italics but only for the related terms mentioned in the description: https://redhat-documentation.github.io/supplementary-style-guide/#action
- Satellite uses single quotations: https://redhat-documentation.github.io/supplementary-style-guide/#capsule-server

In the documentation we use italics only for a definition -not for general emphasis, so I'd vote for that approach across this guide. For this PR though, I'll update it according to the majority of products with double quotations as you suggest @joaedwar.
thanks again :-]

Comment (ndeevy) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc]: Okidoke, @joaedwar have another look please.
I change the entries with two exceptions. Here because it's a definition:

`Description: An _instance_ is a running virtual machine, ..`

and here for the articles:
`If you need to use the indefinite article before "RHOSP", use 'a' not 'an' ..`

Comment (joaedwar) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc]: Thanks @ndeevy for the thorough explanation and for implementing the changes. You are wonderful and so is this PR :]

**Review by joaedwar — APPROVED** [review]

**Comment by joaedwar** [comment]
https://github.com/orgs/redhat-documentation/teams/ccs-style-council Please review (or approve) if you can. Thanks a mil

**Review by bergerhoffer — APPROVED** [review]
LGTM

File: supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc
Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc]: Hi @ndeevy . Would you be OK to incorporate the above suggestion?

@porueesq , you OK to approve this PR after this thread is marked as "Resolved"?

Comment (porueesq) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc]: Sure @dfitzmau . I'll keep my eyes open :eyes:

Comment (ndeevy) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc]: Hi @dfitzmau that addition was made in the **Files changed** tab, I just hadn't hit **Resolve** here so doing that now. Thanks!

Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-openstack-platform.adoc]: Thanks. I had to expand the Files Changes preview to see the link. 

Good work! I'll merge this PR.

**Review by porueesq — APPROVED** [review]

**Pull request merged**
